src/com/projectkorra/projectkorra/waterbending/multiabilities/WaterArmsWhip.java

```diff
@@ -599,11 +584,11 @@ public void setWhipSpeed(final int whipSpeed) {
 	}
 
 	public long getHoldTime() {
-		return this.holdTime;
+		return this.grabDuration;
 	}
 
 	public void setHoldTime(final long holdTime) {

```

Comment:
This method and its parameter should be renamed to better reflect the specific variable it modifies. Something like `setGrabDuration` would be more fitting.